src/main/resources/com/cloudbees/plugins/credentials/CredentialsStoreAction/DomainWrapper/dialog2.jelly

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ ~ The MIT License
+ ~
+ ~ Copyright (c) 2011-2016, CloudBees, Inc., Stephen Connolly.
+ ~
+ ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+ ~ of this software and associated documentation files (the "Software"), to deal
+ ~ in the Software without restriction, including without limitation the rights
+ ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ ~ copies of the Software, and to permit persons to whom the Software is
+ ~ furnished to do so, subject to the following conditions:
+ ~
+ ~ The above copyright notice and this permission notice shall be included in
+ ~ all copies or substantial portions of the Software.
+ ~
+ ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ ~ THE SOFTWARE.
+ -->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form" xmlns:l="/lib/layout">
+    <j:set var="descriptor" value="${app.getDescriptorByName(request2.getParameter('kind'))}"/>
+    <st:setHeader name="X-Wizard-Title" value="Add ${descriptor.displayName}" />
+
+    <l:ajax>
+        <f:form action="${it.store.getRelativeLinkTo(it.domain)}createCredentials" method="POST" name="newCredentials">
+            <div style="margin-top: 0.5rem; display: flex; flex-direction: column; gap: 1.25rem">
+                <div>
+                    <f:rowSet name="credentials">
+                        <j:set var="instance" value="${null}"/>
+                        <f:class-entry descriptor="${descriptor}"/>
+                        <st:include from="${descriptor}" page="${descriptor.configPage}"/>
```

Comment:
Password isn't redacted in username and password:
<img width="583" height="109" alt="image" src="https://github.com/user-attachments/assets/9fcdea84-8f96-48e2-969a-2fb95b28bb01" />